tests/test_objective_funs.py

```diff
@@ -1571,15 +1571,15 @@ def test_linking_current(self):
         obj = LinkingCurrentConsistency(eq, coilset1)
         obj.build()
         f = obj.compute(coilset1.params_dict, eq.params_dict)
-        np.testing.assert_allclose(f, 0)
+        np.testing.assert_allclose(f, 0, atol=1e-8)
```

Comment:
really should have had an atol here before, since default atol is 0 which is super fragile when the target value is also 0